prometheus-exporters/vrops-exporter/alerts/vccluster.alerts

```diff
@@ -0,0 +1,67 @@
+groups:
+- name: vccluster.alerts
+  rules:
+  - alert: VCenterRedundancyLostFailoverHostMissing
+    expr: |
+      count by (vcenter, vccluster) (vrops_hostsystem_configuration_dasconfig_admissioncontrolpolicy_failoverhost{vccluster=~"production.*"} == 1)
```

Comment:
Are we missing some expression here? 
I'd expect `== 0`?

Reply:
Yes. Copied without `== 0`  Will fix.